content/en/docs/reference/_index.md

```diff
@@ -19,7 +21,19 @@ This section of the Kubernetes documentation contains references.
 ## API Reference
 
 * [Kubernetes API Reference {{< latest-version >}}](/docs/reference/generated/kubernetes-api/{{< latest-version >}}/)
-* [Using The Kubernetes API](/docs/reference/using-api/) - overview of the API for Kubernetes.
+* [Kubernetes API Overview](/docs/reference/using-api/) - overview of the API for Kubernetes.
+
+## Standardized Glossary
+
+* [Standardized Glossary](/docs/reference/glossary/) - This glossary is intended to be a comprehensive, standardized list of Kubernetes terminology
+
+## API Access Control
+
+* [API Access Control](/docs/reference/access-authn-authz/) - Overview of how Kubernetes implements and Controls API Access
+
+## Kubernetes Issues and Security
+* [Kubernetes Issue Tracker](/docs/reference/issues-security/issues/)
+* [Kubernetes Security and Disclosure Information](/docs/reference/issues-security/security/)
 
 ## API Client Libraries
```

Comment:
There are 6 official libraries (the existing page is out of date); see https://k8s.io/docs/reference/using-api/client-libraries/#officially-supported-kubernetes-client-libraries

Maybe change “Officially supported client libraries” to a hyperlink, targeting https://k8s.io/docs/reference/using-api/client-libraries/#officially-supported-kubernetes-client-libraries ?

Reply:
@raghvenders . Thanks for rebasing.
Could you change this heading from "API Client Libraries" to "Officially supported client libraries"?